src/main/java/pl/byczazagroda/trackexpensesappbackend/service/WalletServiceImpl.java

```diff
@@ -1,13 +1,30 @@
 package pl.byczazagroda.trackexpensesappbackend.service;
 
 import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
 import org.springframework.stereotype.Service;
+import pl.byczazagroda.trackexpensesappbackend.dto.WalletDTO;
+import pl.byczazagroda.trackexpensesappbackend.mapper.WalletModelMapper;
 import pl.byczazagroda.trackexpensesappbackend.repository.WalletRepository;
 
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Slf4j
 @Service
 @RequiredArgsConstructor
 public class WalletServiceImpl implements WalletService {
 
     private final WalletRepository walletRepository;
 
+    private final WalletModelMapper walletModelMapper;
+
+    @Override
+    public List<WalletDTO> getAllWallets() {
+        log.info("Return list of all wallets");
+        return walletRepository.findAll()
+                .stream()
+                .map(walletModelMapper::mapWalletEntityToWalletDTO)
+                .collect(Collectors.toList());
```

Comment:
Samo .toList wystarczy, nie ma potrzeby dawać Collectors

Reply:
Done, changed used method from collect() to toList()